### PR TITLE
Adding initial dev setup instructions and updating main readme

### PR DESCRIPTION
**Pull request by nzthiago**

### CODE REVIEW

File: docs/CreateAzureADforAzureBot.md [hunk 1]
```diff
@@ -2,19 +2,19 @@
 
 If you would like to authenticate against your own Azure AD for development or private use of the AzureBot code, you'll need to setup your own Azure AD application.  Here are the steps you need to follow:
 
-1. Create or Utilize an Azure Active Directory (AAD) Tenant
-2. Create a new Azure AD app on the AAD tenant
-3. Configure the AAD application for use with the AzureBot
-4. Configure App Multi-Tenancy (optional)
+- [Create or Utilize an Azure Active Directory (AAD) Tenant](#createtenant)
```
Comment (dtzar) [inline docs/CreateAzureADforAzureBot.md]: Why this change from numbering?  Also the links don't work...

File: docs/DeveloperSetup.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+## LUIS Model
+
+AzureBot uses [Language Understanding Intelligent Service (LUIS)](https://www.luis.ai), a part of [Microsoft Cognitive Services](https://www.microsoft.com/cognitive-services/), to understand the user's intent. 
+
+The AzureBot code in this repo has the the ***LuisModel*** attribute on the ***ActionDialog*** class already configured with our LUIS model, so if you are not adding to the bot code you don't need to craete your own LUIS applicaiton.
+
+But if you would like to develop with your own model and extend it with your additions to the bot, or want to understand how the LUIS model works, [create an account with LUIS](https://www.luis.ai). Then create a new application using New App -> Import Existing App. Then, in this new app dialog box, upload the [AzureBot.json file](../AzureBot/LuisModel/AzureBot.json).   
+
+Once you have created the new LUIS application, train and [publish it](https://www.luis.ai/Help/#PublishingModel), then update the ***LuisModel*** attribute in the ***ActionDialog*** class with the new application ID and subscription key. You can get the application ID and subscription key from the LUIS application published URL.
```
Comment (dtzar) [inline docs/DeveloperSetup.md]: Link to ActionDialog file would be nice.

File: docs/UX.Guidelines.md [hunk 3]
```diff
@@ -2,7 +2,6 @@
 This document should help you design or add new functions to AzureBot in a way that is consistent with other behaviors.
 
 
-
```
Comment (dtzar) [inline docs/UX.Guidelines.md]: Removing a line, eh?

File: docs/CreateAzureADforAzureBot.md (hunk 1 above)
Comment (nzthiago) [inline docs/CreateAzureADforAzureBot.md]: I wasn't sure about numbering as they're not all needed, so thought about a list to be consistent. But I can change my dev setup one to add numbering, as long as both match, let me know what you prefer (numbering I suspect :))

File: docs/UX.Guidelines.md (hunk 3 above)
Comment (nzthiago) [inline docs/UX.Guidelines.md]: I appreciate your appreciation of my appreciation for detail

File: docs/CreateAzureADforAzureBot.md (hunk 1 above)
Comment (dtzar) [inline docs/CreateAzureADforAzureBot.md]: I don't believe the formatting needs to be the same as the developer setup guide.  For the AAD guide, the steps DO need to be completed in that order.  For yours if you wanted to make things more clean and similar - then I'd say you could have a pre-requisites section (could be completed in any order) that would include everything up until "Putting all together" and then you could have a numbered list of steps if you wanted.

Comment (nzthiago) [inline docs/CreateAzureADforAzureBot.md]: Ok, I'll reverse the numbering for AAD. I did start with pre-requisites for the dev one, but then most of it turned into a pre-requisite section other than the 'putting it all together' part so ended up leaving it all as part of the higher level.

File: README.md [hunk 4]
```diff
@@ -1,8 +1,33 @@
 # AzureBot
-This is the home of the code used to run the AzureBot and is built with the [Microsoft Bot framework] (http://docs.botframework.com/), [Microsoft Bot Builder C# SDK](http://docs.botframework.com/sdkreference/csharp/), and the [Azure Resource Manager Nuget package](https://www.nuget.org/packages/Microsoft.Azure.Management.ResourceManager).
+The AzureBot was created to improve the productivity of any developer, admin, or team working with Azure. It is not currently publicly available, but you can follow the [Developer Set Up](#developersetup) to run it yourself and contribute.
```
Comment (dtzar) [inline README.md]: developersetup link doesn't work.  I'd either remove the link as-is or just link to the actual developer setup guide as you did below.

File: docs/CreateAzureADforAzureBot.md [hunk 5]
```diff
@@ -35,7 +35,7 @@ Go to the configure tab of the application you just created and do the following
  * Select Delegated Permissions and click the checkbox to "Access Azure Service Management..."
  <br> ![AzureAD-Permissions](media/AzureAD-Permissions.jpg)
 
-## 4. Configure App Multitenancy
+## 4. Configure App Multi-Tenancy (optional)
```
Comment (dtzar) [inline docs/CreateAzureADforAzureBot.md]: Multitenancy IS a single word ;) https://en.wikipedia.org/wiki/Multitenancy

**Comment by nzthiago** [comment]
Ok, have updated links and numbering and table, and fixed the dev set up link. Can you review again?
